assests/javascript/index.js

```diff
@@ -44,6 +51,68 @@ async function addTask(task) {
   }
 }
 
+async function getAllTasks() {
+  try {
+    const response = await fetch(`${baseUrl}/todos`);
+    const {data} = await response.json();
+
+    return sortTasks(data);
+  } catch {
+    showConnecionErrorMassage();
+  }
+}
+
+async function showAllTasks() {
+  const tasks = await getAllTasks();
+
+  taskList.innerHTML = tasks.map((task) => {
+    return `<li class="task-box tooltip" id=${task.id}>
```

Comment:
id value is not inside `""`. Check the DOM after creating a new tasks.

Reply:
yesss fixed